realm/realm-library/src/main/java/io/realm/internal/util/Pair.java

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
To prepare for supporting the JVM where depending on anything in `android.*` will be a no-go.